### ISSUE TITLE
FAB "Add post" keeps confusing our users

### ISSUE THREAD

**Issue by rowasc** [ushahidi/platform#4028]
**Describe the bug**
The round "Add post" button continues to confuse our users. We want to change it to something more explicit. I'm not entirely sure what, but definitely something that at least says "Add" or "Send report" depending on how it looks... and that it looks more like a button since most people have no idea what that round button does until they spend some time looking at the screen.

This is also related to accessibility: https://www.w3.org/TR/WCAG21/#non-text-content
Aha! Link: https://ushahiditeam.aha.io/features/PROD-46

**Comment by Angamanga** [comment]
**Reference docs:**
UI-refresh-issues: https://github.com/ushahidi/platform/labels/Theme%3A%20JS%20Framework%20rewrite

**Comment by Angamanga** [comment]
@rowasc @CeciliaHinga I have done some experimenting with the buttons. These changes can be done fairly easy with small changes in the css + platform-client.

**Map-view-desktop:** https://drive.google.com/file/d/1J6a2l0j_XUnwVLv6a7AoF0NoDgBmcPEA/view?usp=sharing
<img width="1308" alt="Screenshot 2020-08-05 at 13 10 43" src="https://user-images.githubusercontent.com/8624777/89408684-feae9100-d720-11ea-89b4-f6f81d983810.png">

**Map-view-mobile:** https://drive.google.com/file/d/1dDG1xIRmvrQ-So-odN4uNrYDRxSA13wD/view?usp=sharing
<img width="520" alt="Screenshot 2020-08-05 at 13 11 01" src="https://user-images.githubusercontent.com/8624777/89408727-1c7bf600-d721-11ea-96d8-6d1a24f4f4e2.png">


**Data-view-desktop:** https://drive.google.com/file/d/1ljbd8UUnmOZ8uZdQKjGx9yAde1jnKJ3E/view?usp=sharing
<img width="1042" alt="Screenshot 2020-08-05 at 13 15 11" src="https://user-images.githubusercontent.com/8624777/89408750-27368b00-d721-11ea-9736-97be733dd9ef.png">

**Data-view mobile:** https://drive.google.com/file/d/1wvvIVT2MG6QlVH_t4jEs3imBQZZNkuje/view?usp=sharing
<img width="520" alt="Screenshot 2020-08-05 at 13 14 52" src="https://user-images.githubusercontent.com/8624777/89408775-30bff300-d721-11ea-8f40-28e7aa2d9ecc.png">

**Comment by rowasc** [comment]
@Angamanga early feedback from Walter is that the "plus" sign is kind of unnecessary, which I tend to agree with but maybe we're missing something , just sending feedback your way

**Comment by rowasc** [comment]
@Angamanga another thing in my mind is if we can make it so that when you scroll all the way down in a post, the button never overlaps with the post content (so it doesn't block you from viewing what's underneath)

**Comment by rowasc** [comment]
@Angamanga also maybe we can  think of saying "Send a survey response" or "Send a report" instead of "Add a new post"  since reports and responses is closer to the terms we use through the app and the new terms we are trying to use . I don't know if regular users know what a post is tbh

**Comment by Angamanga** [comment]
@rowasc Hm, I totally and completely support changing posts to responses and reports. It makes so much more sense. But that increases the scope of this issue. We use posts everywhere in our docs, on the webpage, on the pricing-page, in the tour, and so on...  Maybe even in our videos. That is a much bigger change than just changing the button. But still, I am 100% FOR making that change and doing it at the same time as changing this button is a good time since it makes the use of the word posts/reports more visible. But it will take longer time than a few lines of code in the client and the pl.

**Comment by CeciliaHinga** [comment]
@rowasc If we are thinking of broadening the scope of this change then your suggestions are good but if we decide not to do that, what about changing it to "submit your post" this way they will know that is a button that needs to be clicked to submit something?

**Comment by rowasc** [comment]
@Angamanga @CeciliaHinga totally agree with ya'll. Let's keep scope contained. I should have started a different ticket. 
Let's use "Submit a post" and then we can separately do the changes I suggested above.

**Comment by rowasc** [comment]
@Angamanga Did we stop this one due to other priorities? Do you want me to add it to the next release target?

**Comment by Angamanga** [comment]
@rowasc Yea, I have started making the changes but reprioritised it. It requires changes in both the PL and the client so I think we should not make it a community-task right now.

**Comment by rowasc** [comment]
Ahhhh ok. Yea makes sense I'll just leave it assigned to you then

**Comment by hariclerry** [comment]
Hello @Angamanga, @rowasc, I'm an outreachy applicant, can I please work on this?

**Comment by rowasc** [comment]
No, but I assigned a different ticket. Thanks!

**Comment by Angamanga** [comment]
@Angamanga Take up work on this again
